website/blog/2023-12-06-0.73-debugging-improvements-stable-symlinks.md

```diff
@@ -0,0 +1,179 @@
+
+#### Min SDK bump
+
+React Native 0.73 will be the final version which supports Android 5.0 (API Level 21). The next version of React Native will have a minimum SDK version of 23 (Android 6.0). Read more about the upcoming changes to min SDK bump [here](/). <!-- TODO -->
+
+### Deprecated Debugging Features
+
+#### Flipper ↔ React Native integration
+
+We are gradually moving away from Flipper as the default tool for debugging React Native apps. In 0.73, this begins by deprecating the native Flipper integration included with React Native (bootstrap code which wires up Flipper's core plugins). We will be removing this integration and dependency in the next release — meaning parts of Flipper such as the Network plugin will cease to work.
+
+**Unchanged**: Flipper as a standalone product for native app debugging will continue to exist. Even after its removal from new React Native projects in future, developers will be able to [manually add Flipper to their app](https://fbflipper.com/docs/getting-started/android-native/) if they wish.
+
+For more information on why we are moving away from Flipper, [view the RFC](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0641-decoupling-flipper-from-react-native-core.md).
+
+#### Remote JavaScript Debugging
+
+Remote JavaScript Debugging is a legacy debugging mode that connects an external web browser (Chrome) to your app and runs your JavaScript code inside a web page, i.e. `http://localhost:8081/debugger-ui`. This model could lead to inconsistent app behaviour while debugging, and is incompatible with native modules under the New Architecture.
+
+In 0.73, Remote JavaScript Debugging is deprecated and has been removed from the Dev Menu. Enabling the remote debugger must now be done manually via the `NativeDevSettings` API. Doing this is covered in the [Other Debugging Methods docs](/docs/next/other-debugging-methods#remote-js-debugging).
+
+:::info
+Remote JavaScript Debugging was previously the default debugging experience for apps using JavaScriptCore (JSC). We recommend [Safari Developer Tools (direct JSC debugging)](/docs/next/other-debugging-methods#safari-developer-tools-direct-jsc-debugging) instead, for iOS apps.
+
+We recommend using [Hermes](/docs/hermes) for a consistent debugging experience on all platforms.
+:::
+
+### New Architecture Updates
+
+We continue active development on the New Architecture and are fully invested in bringing it to the Open Source community.
+
+In 0.73, Bridgeless Mode is now available for experimentation. We're shipping this with a new interop layer to continue support for legacy Native Modules. And we're introducing Static View Configs, which simplify native view components on iOS.
+
+More info on these features will be covered in an incoming blog post.
```

Comment:
```suggestion
We continue the rollout of the New Architecture of React Native, to make it available to everyone in the Open Source community.

Since React Native 0.68, both the New Renderer (Fabric) and the New Native Module System (TurboModule) were available to users to experiment and evaluate. We want to thank the community for the feedback we received so far.

Today we're releasing another piece of the New Architecture: **Bridgeless mode**. Up until now, when you enable the New Architecture is your apps, the Bridge was still available to support backward compatibility with older components and modules. However, our vision is to fully sunset the bridge. Starting from React Native 0.73, you can enable Bridgeless Mode which will disable the creation of the bridge entirely.

Together with Bridgeless Mode, we're also shipping a Native Module Interop Layer, that will allow you to reuse your old modules when in Bridgeless Mode. The [Renderer Interop Layer](https://github.com/reactwg/react-native-new-architecture/discussions/135) introduced in React Native 0.72 has also been adapted to work with Bridgeless Mode. 

As the rest of the New Architecture, Bridgeless Mode is considered experimental. We invite interested users to enable it and report to us problems and incompatibilities you might face in the New Architecture working group. There you can [read more about this in  details and report your feedback](TODO).
```